src/data-display/tags/PTag.vue

```diff
@@ -9,7 +9,11 @@
              width="0.8rem"
              height="0.8rem"
         />
-        <span class="text"><slot /></span>
+        <slot name="default">
+            <span v-if="categoryItem" class="category"><slot name="category">[{{ categoryItem.name || categoryItem.label }}]</slot></span>
```

Comment:
```suggestion
            <span v-if="categoryItem" class="category"><slot name="category">[{{ categoryItem.label || categoryItem.name }}]</slot></span>
```

I think that `label` has higher priority than `name`.